Update steps to account for cross-namespace Argo CD permission cleanup (#35854)
openshift/openshift-docs: modules/gitops-creating-an-application-by-using-the-oc-tool.adoc

----
$ oc get application -n openshift-gitops
----

> . Add a label to the namespace your application is deployed in so that the Argo CD instance in the `openshift-gitops` namespace can manage it:

Reviewer: ```suggestion
. Add a label to the namespace where your application is deployed. This way, the Argo CD instance in the `openshift-gitops` namespace can manage your <thing>:
```
Does separating this into two sentences make it easier to read? What does "it" refer to?

Author: the "it" is referring to the namespace where your application is deployed in... so how would I say that without sounding repetitive?

Reviewer: Seems okay.

+
[source,terminal]
----

> $ oc label namespace spring-petclinic argocd.argoproj.io/managed-by=openshift-gitops

Reviewer 2: Request you to please check the steps above this one and verify that this command will work with this section: https://deploy-preview-35854--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html#creating-an-application-by-using-the-oc-tool_configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations
Namespace in this YAML: https://github.com/redhat-developer/openshift-gitops-getting-started/blob/main/argo/cluster.yaml which is mentioned in the step 2 of this section is `Default`. 
So is `spring-petclinic argocd.argoproj.io` the default namespace? If yes, we might want to mention that as a note, below the 2nd step.
Or we can change this command to : `$ oc label namespace <application_namespace>/managed-by=openshift-gitops`

Reviewer 3: > So is spring-petclinic argocd.argoproj.io the default namespace? If yes, we might want to mention that as a note, below the 2nd step.

The namespace in this case is `spring-petclinic`, the `argocd.argoproj.io` is part of the label (the name prefix), and should stay untouched.
